esrally/reporter.py

```diff
@@ -292,6 +294,13 @@ def _report_transform_stats(self, stats):
 
         return lines
 
+    def _report_ingest_pipeline_stats(self, stats):
+        return self._join(
+            self._line("Total Ingest Pipeline count", "", stats.ingest_pipeline_cluster_count, ""),
+            self._line("Total Ingest Pipeline time", "", stats.ingest_pipeline_cluster_time, "ms"),
```

Comment:
```
|  Total Ingest Pipeline time |   | 2.56823e+07 |     ms |
```

Should we use another metric than ms here? I guess it was only so big in this case because I ran a big track